tests/infra/testmode.py

```diff
@@ -0,0 +1,12 @@
+import os
+import unittest
+
+
+def standalone(f):
+    return unittest.skipIf(
+        ("INTEGRATION_TESTS" in os.environ and os.environ['INTEGRATION_TESTS'] != "0"),
+        "Only running integration tests")(f)
+
+
+def integration(f):
+    return unittest.skipUnless("INTEGRATION_TESTS" in os.environ, "Only running integration tests")(f)
```

Comment:
The double negation is making my head explode. But if I am correct, this will run integration tests if `INTEGRATION_TESTS` is `0`. Is that what you want?

s/integration/standalone in message?